transactions/adminSetFindCut.cdc

```diff
@@ -1,10 +1,40 @@
 import Admin from "../contracts/Admin.cdc"
+import FindMarket from "../contracts/FindMarket.cdc"
+import FlowToken from "../contracts/standard/FlowToken.cdc"
+import FUSD from "../contracts/standard/FUSD.cdc"
+import FiatToken from "../contracts/standard/FiatToken.cdc"
+import DapperUtilityCoin from "../contracts/standard/DapperUtilityCoin.cdc"
+import FlowUtilityToken from "../contracts/standard/FlowUtilityToken.cdc"
+import MetadataViews from "../contracts/standard/MetadataViews.cdc"
+import FungibleToken from "../contracts/standard/FungibleToken.cdc"
 
-transaction(tenant: Address, saleItemName: String, cut: UFix64){
+transaction(tenant: Address, cut: UFix64){
     prepare(account: AuthAccount){
         let adminRef = account.borrow<&Admin.AdminProxy>(from: Admin.AdminProxyStoragePath) ?? panic("Cannot borrow Admin Reference.")
 
-        adminRef.setFindCut(tenant: tenant, saleItemName:saleItemName, cut: cut, rules: nil, status: "active")
+		let defaultRules : [FindMarket.TenantRule] = [
```

Comment:
The Cut rules are now constructed in transaction and I put some rules by default